### PR TITLE
add prior microformats measure proposals

### PR DESCRIPTION
**Pull request by tantek**
add prior microformats measure proposals, linking to overall research, hmeasure proposal, updated h-measure proposal

### CODE REVIEW

File: amount.md [hunk 1]
```diff
@@ -43,4 +43,10 @@ however without any default user-discernible effects,
 including [in screen readers](https://twitter.com/LeonieWatson/status/1333078194925264898), 
 despite the “[First rule of ARIA use](https://www.w3.org/TR/aria-in-html/#rule1)” encouraging publishers to use native HTML semantic elements.
 
+The microformats community did a lot of [research into measure formats](https://microformats.org/wiki/measure) 
+for amounts with physical units in the mid-2000s, 
+culminating in an [hmeasure draft proposal](https://microformats.org/wiki/measure-brainstorming#Draft_Schema), 
+which was eventually evolved in the 2010s into an [h-measure proposal](https://microformats.org/wiki/measure-brainstorming#microformats2)
```
Comment (martinthomson) [inline amount.md:49]: That's interesting, but a gram is a measure of mass, not weight.

**Review by martinthomson — COMMENTED** [review]
Do you have any opinion about the microformats stuff: what was learned, what might be improved, etc..?  Just saying "this is prior art" is useful, but the goal here should be to move the ball forward and it's not clear what you are intending without expressing some opinions.

File: amount.md (hunk 1 above)
Comment (tantek) [reply amount.md:49]: While technically true, in context (locale to locale — on Earth), and in practice (automatic unit conversion UIs in e.g. iOS, if you type in 2.2lbs into Notes app, it puts a grey underline on it, and when tapped, shows a pop-up with "Grams 997.9 g, Kilograms 1 kg, Ounces 35.2 oz, Pounds 2.2 lb") the identification and conversion of such units is what users expect, not physics theoretical purity. 
Also, that's a critique of a cited resource, not of this PR so I'm not sure what the relevance is unless you consider it a critique severe enough to not link to the citation?

**Comment by tantek** [comment]
> Do you have any opinion about the microformats stuff: what was learned, what might be improved, etc..? Just saying "this is prior art" is useful, but the goal here should be to move the ball forward and it's not clear what you are intending without expressing some opinions.

Yes I have both opinions and additional information about the uses and adoption in particular contexts (e.g. recipes) however I didn't think that was necessary as part of this incremental improvement to this Explainer. They can be landed in future incremental PRs.

I'd rather develop a culture and habit of enabling rapid incremental updating / expanding of Explainers, as one would edit a wiki page or a Google Doc, instead of some unknown larger "chunk" size that Explainer authors/editors are expected to produce in each PR, which I believe would be counter-productive and directly in opposition to our goals of rapidly developing minimum viable explainers, and then rapidly incrementally growing them within the sections noted in the README.

**Pull request merged**